stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -146,10 +153,9 @@ open class StripeEditText @JvmOverloads constructor(
     }
 
     private fun determineDefaultErrorColor() {
-        cachedColorStateList = textColors
         defaultErrorColor = ContextCompat.getColor(
             context,
-            if (StripeColorUtils.isColorDark(textColors.defaultColor)) {
```

Comment:
Here we used textColors without doing a null check, so I think it is ok to continue to have cachedColorStateList as non-nullable.

Reply:
If we do want to make it nullable, then we just need to figure out what do for the default color in this case.